analysis/topEFT/topeft.py

```diff
@@ -158,16 +190,19 @@ def process(self, events):
           elif(self.jetSyst == 'JESDown'):
             jets = corrected_jets.JES_jes.down
           '''
-        j['isGood']  = isTightJet(getattr(j, jetptname), j.eta, j.jetId, j.neHEF, j.neEmEF, j.chHEF, j.chEmEF, j.nConstituents)
-        #j['isgood']  = isGoodJet(j.pt, j.eta, j.jetId)
-        #j['isclean'] = isClean(j, e, mu)
-        j['isClean'] = isClean(j, e, drmin=0.4)& isClean(j, mu, drmin=0.4)# & isClean(j, tau, drmin=0.4)
-        goodJets = j[(j.isClean)&(j.isGood)]
+
+        cleanedJets['isGood']  = isTightJet(getattr(cleanedJets, jetptname), cleanedJets.eta, cleanedJets.jetId, jetPtCut=25.) # temporary at 25 for synch
+        goodJets = cleanedJets[cleanedJets.isGood]
+        
+        
+        
+        
+        # count jets, jet 
         njets = ak.num(goodJets)
         ht = ak.sum(goodJets.pt,axis=-1)
         j0 = goodJets[ak.argmax(goodJets.pt,axis=-1,keepdims=True)]
-        #nbtags = ak.num(goodJets[goodJets.btagDeepFlavB > 0.2770])
-        # Loose DeepJet WP
+        
+        # to do: check these numbers are ok
         if year == 2017: btagwpl = 0.0532 #WP loose 
         else: btagwpl = 0.0490 #WP loose 
         isBtagJetsLoose = (goodJets.btagDeepB > btagwpl)
```

Comment:
I believe these are mine, and they should be correct for 2017. However, maybe we should consider a JSON file for these, instead of hard coding them.